indico/modules/search/templates/category_search.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'layout/base.html' %}
+
+{% block page_class %}search-page{% endblock %}
+
+{% block title %}
+    {% trans %}Searching within{% endtrans %} "{{ category.title }}"
```

Comment:
```suggestion
    {% trans cat=category.title %}Searching within "{{ cat }}"{% endtrans %}
```

not sure if all languages put it at the end